packages/unchained-client/src/cosmossdk/parser/index.ts

```diff
@@ -103,18 +103,25 @@ export class BaseTransactionParser<T extends Tx> {
         })
       }
 
+      const isSpecialDepositEvent = (() => {
```

Comment:
going to look at moving this logic to the backend where it is more straightforward to parse